sass/edx/_variables.scss

```diff
@@ -645,22 +645,21 @@ $nav-tabs-justified-active-link-border-color: transparent !default;
 //
 // Define colors for form feedback states and, by default, alerts.
 
-$state-success-text:             $green !default;
-$state-success-bg:               $body-bg !default;
-$state-success-border:           $green !default;
+$state-success-text:               $black !default;
+//$state-success-bg:               #dff0d8 !default;
```

Comment:
I had the same thought. I hate this manual indenting, so I'm tempted to remove all of the spaces, as I don't think it really helps. For now, I'm going to merge as is, but let me know what you think.